hal/interfaces/Gpio.hpp

```diff
@@ -5,14 +5,14 @@
 
 namespace hal
 {
-    enum class InterruptTrigger: uint8_t
+    enum class InterruptTrigger : uint8_t
```

Comment:
Can clang-format put the ": uint8_t" on the next line?

Reply:
Not that I know off; it even has a preference of putting things on one line. Like in the inheritance case.